recipes/regiis.rb

```diff
@@ -28,4 +28,5 @@
   command "%WINDIR%\\Microsoft.Net\\Framework64\\v4.0.30319\\aspnet_regiis.exe -i -enable > #{guard_file}"
   creates guard_file
```

Comment:
Ok, maybe the documentation is not clear enough.
Take a look at the implementation: https://github.com/chef/chef/blob/master/lib/chef/provider/execute.rb#L51

If the file is present the execute resource does nothing.

What make you say that the execute resource run at every chef-client runs? Could you provide a chef-client output output?